env/work/srw_python/SRWLIB_Example15.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
```

Comment:
The data files (`intensity_0.0m.dat`, `intensity_3.0m.dat` and `intensity_3.9m.dat`) are saved in `data_example_15/` folder. If Matplotlib is not available, they can be used for further plotting by a third-party program.